### PR TITLE
fix(.env): .env files formatting

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **fix**

### Description

this PR updates some files to match our current .env.shared formatting.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

### Changelog updates

#nochangelog
#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,3 +1,5 @@
+insert_final_newline = true
```
Comment (pvinis) [inline .editorconfig:1]: a nice little thing for all files being "clean"

File: .env.example [hunk 2]
```diff
@@ -1,18 +1,41 @@
+ARTSY_API_CLIENT_KEY=-
```
Comment (pvinis) [inline .env.example:1]: formatting

File: .editorconfig (hunk 1 above)
Comment (gkartalis) [reply .editorconfig:1]: 😍 love this

File: package.json [hunk 3]
```diff
@@ -46,6 +46,7 @@
     "relay": "relay-compiler",
     "setup:artsy": "./scripts/setup-env-for-artsy",
     "setup:oss": "./scripts/setup-env-for-oss",
+    "setup:artsy:update!": "scripts/update-env-for-artsy",
```
Comment (pvinis) [inline package.json:49]: woop, new script to upload changed to .env.shared.

File: src/setupJest.ts [hunk 4]
```diff
@@ -542,7 +542,6 @@ jest.mock("react-native-config", () => ({
   SENTRY_DSN: "sentry_dsn",
   GOOGLE_MAPS_API_KEY: "google_maps_api_key",
   MAPBOX_API_CLIENT_KEY: "mapbox_api_client_key",
-  SAILTHRU_KEY: "sailthru_key",
```
Comment (pvinis) [inline src/setupJest.ts:545]: 👋

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ✅ **No changelog changes**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5a9a910ac85f842e695ffca6113120758e0654c1
</p>

File: .env.example [hunk 5]
```diff
@@ -1,18 +1,41 @@
...
 GOOGLE_MAPS_API_KEY=-
 MAPBOX_API_CLIENT_KEY=-
-MAPBOX_DOWNLOAD_TOKEN=-
 SEGMENT_PRODUCTION_WRITE_KEY_ANDROID=-
 SEGMENT_PRODUCTION_WRITE_KEY_IOS=-
 SEGMENT_STAGING_WRITE_KEY_ANDROID=-
 SEGMENT_STAGING_WRITE_KEY_IOS=-
 SENTRY_DSN=-
-SENTRY_STAGING_DSN=-
+SPLIT_IO_PRODUCTION_API_KEY=-
```
Comment (araujobarret) [inline .env.example:17]: do we still use split io?

File: .editorconfig (hunk 1 above)
Comment (brainbicycle) [reply .editorconfig:1]: oh nice that always gets me

**Review by brainbicycle — APPROVED** [review]

**Comment by brainbicycle** [comment]
looks great, minor request, can we add a type and description to the pr?

**Pull request merged**